test/parallel/test-child-process-validate-stdio.js

```diff
@@ -5,14 +5,13 @@ const common = require('../common');
 const assert = require('assert');
 const getValidStdio = require('internal/child_process').getValidStdio;
 
-const expectedError =
-  common.expectsError({ code: 'ERR_INVALID_OPT_VALUE', type: TypeError }, 2);
+const expectedError = { code: 'ERR_INVALID_OPT_VALUE', type: TypeError };
 
 // Should throw if string and not ignore, pipe, or inherit
-assert.throws(() => getValidStdio('foo'), expectedError);
+common.expectsError(() => getValidStdio('foo'), expectedError);
```

Comment:
Can you please leave it as `assert.throws()` here and elsewhere? Butter to use the API everyone already knows than needlessly replace it with an idiosyncratic internal API.

Reply:
Oh, I see, this just moves the expectsError, rather than introducing it. Never mind my comment. Should eventually be removed anyway but not necessarily in this PF and it probably needs to wait for Node.js 8 to EOL.